modules/related-posts/jetpack-related-posts.php

```diff
@@ -1617,6 +1617,7 @@ protected function _enabled_for_request() {
 		$enabled = is_single()
 			&& ! is_attachment()
 			&& ! is_admin()
+			&& ! is_embed()			
```

Comment:
Could you remove those trailing tabs?

```suggestion
			&& ! is_embed()
```